package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "broccoli-output-wrapper",
-  "version": "3.0.0",
+  "version": "3.1.0",
```

Comment:
FWIW, it's generally better to leave the version number until you are ready to actually publish. That's my process at least...

Reply:
I should also follow that too. I am simply bumping versions that I don't like. I will make a note of this.